File: JamFix.Services/Database/RadniNalog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JamFix.Services.Database
{
    public class RadniNalog
    {
        public int NalogId { get; set; }
        public string OpisPrijavljenog { get; set; }
        public string OpisUradjenog { get; set; }
        public string ImePrezime { get; set; }
        public string Telefon { get; set; }
        public DateTime Datum { get; set; }
        public string Adresa { get; set; }
        public string Mjesto { get; set; }
        public string Naziv { get; set; }
        public int Kolicina { get; set; }
        public int RadnikId { get; set; }
        public Radnik Radnik { get; set; }
    }
}

File: RS2/UI/jamfix_mobilna/jamfix_mobilna/lib/models/radni_nalog.dart
import 'package:json_annotation/json_annotation.dart';

part 'radni_nalog.g.dart';

@JsonSerializable()
class RadniNalog {
  int? nalogId;
  String? nosilacPosla;
  String? opisPrijavljenog;
  String? opisUradjenog;
  String? imePrezime;
  String? telefon;
  DateTime? datum;
  String? adresa;
  String? mjesto;
  String? naziv;
  int? kolicina;
  RadniNalog({this.nalogId,
      this.nosilacPosla,
      this.opisPrijavljenog,
      this.opisUradjenog,
      this.imePrezime,
      this.telefon,
      this.datum,
      this.adresa,
      this.mjesto,
      this.naziv,
      this.kolicina});

  factory RadniNalog.fromJson(Map<String, dynamic> json) =>
      _$RadniNalogFromJson(json);

  Map<String, dynamic> toJson() => _$RadniNalogToJson(this);
}
